README.md

```diff
@@ -1,16 +1,23 @@
-# UEFI for PINE64 Quartz64 (Rockchip RK356X)
+# UEFI for Rockchip RK356X based SBR
 
-This repository contains a port of Tianocore EDK II to the PINE64 Quartz64 board.
+This repository contains a port of Tianocore EDK II to various Rockchip RK356x based boards:
 
-https://www.pine64.org/quartz64a/
+* [PINE64 Quartz64 A](https://www.pine64.org/quartz64a/) / [PINE64 Quartz64 B](https://www.pine64.org/quartz64b/) / [PINE64 SOQuartz](https://wiki.pine64.org/wiki/SOQuartz)
```

Comment:
Lets match the vendor board naming, like `Quartz64 A` -> `Quartz64 Model A`, etc.

Reply:
ack